tests/multi_tenant/conftest.py

```diff
@@ -63,3 +52,25 @@ def mock_openid_ok_then_empty(respx_mock, mock_openid):
 def mock_openid_and_no_valid_keys(respx_mock, mock_openid):
     respx_mock.get(keys_url(version=2)).respond(json=build_openid_keys(no_valid_keys=True))
     yield
+
+
+def generate_obj(issuer=None):
```

Comment:
Could we rename this from `generate_obj` to something a bit more clear? `def generate_azure_scheme_multi_tenant_object` or something?

This test suite is already a bit complex, so I want the code to be as clear as possible.  It is really important for maintainability that the tests are easy to read and navigate.